### ISSUE TITLE
Read Json in chunks

### ISSUE THREAD

**Issue by parasml** [awslabs/aws-data-wrangler#235]
Hi @igorborgest,

I am reading my JSON file in chunks as it is too big in size, In below code.

df = wr.s3.read_json(path1, chunksize=2, lines=True)

df type return is Generator.

I am struggling to save above chunk (df) to parquet format, please share your thoughts on how to achieve this or if we can save it to dataframe.

Thanks,
Prashan

**Comment by parasml** [comment]
Hi @igorborgest 

Thanks for all the help in past.

Just tried with below code, but still not able to read json lines in for loop. Please suggest.

code:
--------
df = wr.s3.read_json(path1, chunksize=5, lines=True)
i = 0
for row in df:
    print("row = ", row)
    
    print("row = ", json.loads(row))
    print("--------------")

=======================================

Regards,
Prashan

**Comment by igorborgest** [comment]
Hi @parasml,

Actually I found a bug that I.ve just fixed in the json reading with chunksize.

Do you mind to test our development branch?

`pip install git+https://github.com/awslabs/aws-data-wrangler.git@dev`

Example:

```py3
for df in wr.s3.read_json(paths, lines=True, chunksize=1):
    print(df)
```

I would like to make sure that it will be fixed on our next version `1.2.0`. Thanks!

**Comment by igorborgest** [comment]
Released on version [`1.2.0`](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.2.0)
